### PR TITLE
Build/10297 automate extension release

### PR DESCRIPTION
**Pull request by hurtadosanti**
https://hivemq.kanbanize.com/ctrl_board/22/cards/10297

### CODE REVIEW

File: src/hivemq-extension/prometheusConfiguration.properties [hunk 1]
```diff
@@ -12,14 +12,10 @@
...
 #
 # -------------------------------------------------------------------------
-
-
 # The ip where the servlet will be hosted
 ip=0.0.0.0
-
 # The port where the servlet will work on
 port=9399
-
```
Comment (YannickWeber) [inline src/hivemq-extension/prometheusConfiguration.properties:22]: Why did you change this?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -2,6 +2,8 @@
 
 Welcome to the HiveMQ Community!
 Glad to see your interest in contributing to HiveMQ Prometheus Extension.
-Please checkout our [Contribution Guide](https://github.com/hivemq/hivemq-community/blob/master/CONTRIBUTING.adoc) to make sure your contribution will be accepted by the HiveMQ team.
+Please checkout our [Contribution Guide](https://github.com/hivemq/hivemq-community/blob/master/CONTRIBUTING.adoc) to
+make sure your contribution will be accepted by the HiveMQ team.
 
-For information on how the HiveMQ Community is organized and how contributions will be accepted please have a look at our [HiveMQ Community Repo](https://github.com/hivemq/hivemq-community).
```
Comment (YannickWeber) [inline CONTRIBUTING.md:7]: Why did you change this?

File: src/hivemq-extension/prometheusConfiguration.properties (hunk 1 above)
Comment (hurtadosanti) [reply src/hivemq-extension/prometheusConfiguration.properties]: I remove the commit

File: CONTRIBUTING.md (hunk 2 above)
Comment (hurtadosanti) [reply CONTRIBUTING.md]: I remove the commit

**Review by YannickWeber — APPROVED** [review]
